src/Controller/EnMarche/ReferentController.php

```diff
@@ -506,13 +507,30 @@ public function jecouteSurveyStatsDownloadAction(Survey $survey, StatisticsExpor
 
     /**
      * @Route("/mon-equipe", name="app_referent_organizational_chart")
+     * @Method({"GET", "POST"})
```

Comment:
Can you use ` methods={"GET|POST"}` in the `@Route` annotations instead ?

Reply:
Will not be a problem anymore with #3556 :+1: